sdk/batch/batch/README.md

```diff
@@ -1,121 +1,127 @@
 ## Azure BatchServiceClient SDK for JavaScript
 
-This package contains an isomorphic SDK (runs both in Node.js and in browsers) for BatchServiceClient.
+This package contains an isomorphic SDK for BatchServiceClient.
 
 ### Currently supported environments
 
-- [LTS versions of Node.js](https://nodejs.org/about/releases/)
-- Latest versions of Safari, Chrome, Edge and Firefox.
+- Node.js version 6.x.x or higher
+- Browser JavaScript
 
-### Prerequisites
+### How to Install
 
-You must have an [Azure subscription](https://azure.microsoft.com/free/).
-
-### How to install
+```bash
+npm install @azure/batch
+```
 
-To use this SDK in your project, you will need to install two packages.
+### How to use
 
-- `@azure/batch` that contains the client.
-- `@azure/identity` that provides different mechanisms for the client to authenticate your requests using Azure Active Directory.
+#### nodejs - Authentication, client creation and list application as an example written in TypeScript.
 
-Install both packages using the below command:
+##### Install @azure/ms-rest-nodeauth
 
 ```bash
-npm install --save @azure/batch @azure/identity
+npm install @azure/ms-rest-nodeauth
 ```
 
-> **Note**: You may have used either `@azure/ms-rest-nodeauth` or `@azure/ms-rest-browserauth` in the past. These packages are in maintenance mode receiving critical bug fixes, but no new features.
-> If you are on a [Node.js that has LTS status](https://nodejs.org/about/releases/), or are writing a client side browser application, we strongly encourage you to upgrade to `@azure/identity` which uses the latest versions of Azure Active Directory and MSAL APIs and provides more authentication options.
+##### Authentication
 
-### How to use
+1. Use `AzureCliCredentials` exported from `@azure/ms-rest-nodeauth`.
```

Comment:
What is the benefit of showing the Azure CLI option over the API key one? Can we skip this and showcase the API key scenario as that is what is shown in the official samples as well?

Reply:
This was the readme used earlier, so I kept them to show case how to use nodeauth. To address your feedback, I deleted the CLI one and kept the shared key and the other nodeauth one.